frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.tsx

```diff
@@ -191,11 +204,27 @@ function ActionCreator({
   );
 }
 
+function maybeGetModelId(
+  state: State,
+  { action, actionId }: OwnProps & ActionLoaderProps,
+) {
+  const isNewAction = !actionId;
+  if (isNewAction) {
+    return null;
+  }
+  return action?.model_id;
```

Comment:
nit: why not just `action?.model_id` without other checks? if `actionId` is undefined `action` won't be loaded anyway

Reply:
Think I wanted to make it explicit there's no model for a new action, but I think you're right and we should clean it up 👍